go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/CosmWasm/wasmd
 
-go 1.15
+go 1.16
```

Comment:
You cannot set go 1.16.8 here?

Reply:
It is not possible to define the patch version here. I got: ` invalid go version '1.16.8': must match format 1.23`.
But I have added the `1.16.8` version to the readme